### ISSUE TITLE
"Why RChain (In Layman's Terms)?" Mini Series

### ISSUE THREAD

**Issue by ICA3DaR5** [rchain/bounties#920]
This issue was created on behalf of @Cryptovideos 

### Benefit to RChain

A series of short animated productions explaining what RChain is about for the Layman. The series will help spread the word of what RChain is for the average person. Non-technical examples of what and how it works.

### Budget and Objective

Objective: See **Benefit to RChain** above.

_Estimated Budget of Task:_ **$[600]**
_Estimated Timeline Required to Complete the Task:_ **[7 days]**
_How will we measure completion?_ **[The video will get uploaded to YouTube and posted here.]**

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by TrenchFloat** [comment]
I'm excited to see this!

**Comment by Valentine-Mario** [comment]
Cool. Really love to see it.

**Comment by Cryptovideos** [comment]
Hereweith the link :https://youtu.be/e8Sx4mgtcWM

**Comment by ICA3DaR5** [comment]
@Cryptovideos Legendary!

**Comment by AyAyRon-P** [comment]
@ICA3DaR5 is this completed? Can we close this out and vote?

**Comment by Cryptovideos** [comment]
Thank you Ayay yes

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ICA3DaR5** [comment]
@AyAyRon-P Yes and yes!
